src/installer/corehost/cli/hostpolicy/hostpolicy_context.cpp

```diff
@@ -42,6 +42,27 @@ namespace
 
         return bundle::runner_t::app()->probe(file_path, offset, size);
     }
+
+#if defined(NATIVE_LIBS_EMBEDDED)
+    extern "C" const void* GlobalizationResolveDllImport(const char* name);
```

Comment:
The globalization import is not used at this time...

Reply:
right, this is no longer needed since Globalization is handled by the default overrider.